src/main/java/recipeditor/command/DeleteCommand.java

```diff
@@ -11,7 +11,7 @@ public class DeleteCommand extends Command {
             + '\n' + "\t /delete -t <recipe_title>";
     private static final String COMMAND_FUNCTION =
             "Delete the recipe of given title or index from recipeditor.";
-    public static final String CORRECT_FORMAT = "The input should be '/delete <flag> <argument>.'";
+    public static final String CORRECT_FORMAT = "The input should be '/delete <recipe flag> <argument>.'";
```

Comment:
Not very precise to say `<recipe flag>` here, as in `EditCommand`, `<recipe flag>` mean `-t`,`-d`,`-i`,`-s`, but here its only `-id` or `-t`